tests/flow/arith/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -162,7 +162,7 @@ let tests = [
     x + \\"\\"; // error
     \\"\\" + x; // error
     x + {}; // error
-    ({}) + x; // error
+    (({}) + x); // error
```

Comment:
Yeah, I noticed it when I was updating the tests and I was clearly unsure on how to deal with it without making it over complex!